olm/README.md

```diff
@@ -0,0 +1,34 @@
+# instructions on how to manually test the olm integration
+
+Get the quay token
+
+```shell
+AUTH_TOKEN=$(curl -sH "Content-Type: application/json" -XPOST https://quay.io/cnr/api/v1/users/login -d '
+{
+    "user": {
+        "username": "'"${QUAY_USERNAME}"'",
+        "password": "'"${QUAY_PASSWORD}"'"
+    }
+}' | jq -r '.token')
+```
+
+validate the olm CSV
+
+```shell
+operator-courier verify olm/olm-catalog/
+operator-courier verify olm/olm-catalog/ --ui_validate_io
+```
+
+go to this [site](https://operatorhub.io/preview) to visually validate the result
+
+push the catalog to the quay application registry
+
+```shell
+operator-courier push olm/olm-catalog/ raffaelespazzoli cert-utils-operator 0.0.1 "${AUTH_TOKEN}"
```

Comment:
Did you intend to use your username?

Reply:
yes, I never got the cop quay account credentials, so I had to use mine.